trade-finance-manager-ui/server/controllers/utilisation-reports/add-payment/index.ts

```diff
@@ -0,0 +1,49 @@
+import { Response } from 'express';
+import { SelectedFeeRecordDetails, SelectedFeeRecordsDetails, getFormattedCurrencyAndAmount, getFormattedReportPeriodWithLongMonth } from '@ukef/dtfs2-common';
+import { AddPaymentViewModel, SelectedReportedFeeViewModel } from '../../../types/view-models';
+import api from '../../../api';
+import { asUserSession } from '../../../helpers/express-session';
+import {
+  getFeeRecordIdFromPremiumPaymentsCheckboxId,
+  getPremiumPaymentsCheckboxIdsFromObjectKeys,
+} from '../../../helpers/premium-payments-table-checkbox-id-helper';
+import { CustomExpressRequest } from '../../../types/custom-express-request';
+
+export type AddPaymentRequest = CustomExpressRequest<{
+  reqBody: object;
```

Comment:
I think this type should probably be `Record<FeeRecordViewModelCheckboxId, 'on'>` (`object` as a type is a bit too broad I think)